org.civicrm.reqangular/test/services/format-currency.spec.js

```diff
@@ -30,21 +30,22 @@ define([
     var testMap = [
       // [thousandSeparator, decimalSeparator, inputAmount, outputAmount]
       // Common cases
+      [',', '*', 0.0, '0'],
       [',', '*', 12345675, '12,345,675'],
       [',', '*', 123456.75, '123,456*75'],
       [',', '*', '123456.7.5', '12,345,675'],
       [',', '*', '12345675', '12,345,675'],
       [',', '*', '12,345,675', '12,345,675'],
       [',', '*', '00,456,75*76', '45,675*76'],
       [',', '*', '12345675*76', '12,345,675*76'],
-      [',', '*', '12,34,56*75*76', '123,456*7576'],
```

Comment:
Why has this been amended? Was the expectation wrong or was the test failing?

Reply:
The code was now amended to format the amount with 2 numbers after decimal separator. So, it's been changed.

Reply:
@bmnepali it might be then needed to 

- doublecheck that we have either whole numbers OR fixed(2) numbers
- understand what is the operation for exceeding decimals: floor, round or ceil
- add more edge cases for 3 digits: 1234.123    1234.128      1234.1

Reply:
Ok! its confirmed with @davialexandre:
> If you try to insert/update a value with more than 2 decimal digits, MySQL will simply ignore the additional ones and will only save the first two

![peek 2017-11-01 13-11](https://user-images.githubusercontent.com/6307362/32315577-c5239734-bfd4-11e7-83fb-5ad36fecc053.gif)

So, if the user inputs the value like 123,345.845 it will be converted into 123,345.84, simply ignoring the additional numbers by only keeping the first two numbers after decimal.